src/status_im/chat/models/message_list.cljs

```diff
@@ -44,18 +46,18 @@
   We check the time, and the author"
   [a b]
   (and
-   (not (:system-message? a))
-   (not (:system-message? b))
-   (= (:from a) (:from b))
-   (<= (js/Math.abs (- (:whisper-timestamp a) (:whisper-timestamp b))) group-ms)))
+    (not (:system-message? a))
+    (not (:system-message? b))
+    (= (:from a) (:from b))
+    (<= (js/Math.abs (- (:whisper-timestamp a) (:whisper-timestamp b))) group-ms)))
```

Comment:
[Nitpick] For math oriented code in Lisps we should avoid one-liners in many occasions. In the example below, anyone can quickly spot that `group-ms` is the second argument of `<=`, whereas in the original code it's not clear if it's perhaps related with the call to `Math.abs`.

Also worth pointing out that as of Clojure 1.11 the core function `abs` should exist in Clojure(Script). We should migrate from 1.10 to 1.11.

```clojure
(<= (js/Math.abs (- (:whisper-timestamp a) (:whisper-timestamp b)))
    group-ms)
```

Reply:
Thanks for this information. This code is not part of the PR. This is unintended formatting changes caused by Zprint